### PR TITLE
Travis-CI integration.

### PR DESCRIPTION
**Pull request by resurtm**
Demo builds: https://travis-ci.org/resurtm/with-related-behavior

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 WithRelatedBehavior
 ===================
 
+[![Build Status](https://secure.travis-ci.org/yiiext/with-related-behavior.png?branch=master)](https://travis-ci.org/resurtm/with-related-behavior)
```
Comment (cebe) [inline readme.md]: the link points to your repo.

File: readme_ru.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 WithRelatedBehavior
 ===================
 
+[![Build Status](https://secure.travis-ci.org/yiiext/with-related-behavior.png?branch=master)](https://travis-ci.org/resurtm/with-related-behavior)
```
Comment (cebe) [inline readme_ru.md]: the link points to your repo.

File: readme.md (hunk 1 above)
Comment (resurtm) [inline readme.md]: Thanks! Fixed.

File: readme_ru.md (hunk 2 above)
Comment (resurtm) [inline readme_ru.md]: Fixed too.

**Comment by cebe** [comment]
Thanks!

**Pull request merged**